.github/workflows/tests.yml

```diff
@@ -21,5 +21,7 @@ jobs:
         - name: Install dependencies
           run: |
             pip install -r code/requirements.txt -r code/dev-requirements.txt -r code/backend/requirements.txt
-        - name: Run Python tests
-          run: cd code/; python -m pytest -m "not azure"
+        - name: Run Unit tests
+          run: cd code/; python -m pytest -m "not azure and not functional"
```

Comment:
Could we use `working-directory` instead of `cd`?

Reply:
Yes good idea!